src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -7,12 +7,21 @@
 
 public class Navigate extends BasePageObject {
 
+  /**
+   * Adding default testing group is invoked explicitly in URL as temporary change until
+   * the end of experiment (01.07.2016) described in DAT-4483.
+   */
+
+  private static final String defaultTestingGroup =
+      "&ABTEST.MERCURY_VIEWABILITY_EXPERIMENT=DEFAULT";
+
   public Navigate toPage(String pageName) {
     String host = UrlBuilder.getHostForWiki();
     String cacheBuster = pageName.equals("") ||
                          pageName.equals("/") ? "" : "?cb=" + DateTime.now().getMillis();
+    String abGroup = pageName.equals("") || pageName.equals("/") ? "" : defaultTestingGroup;
```

Comment:
if you will navigate to page with `/` you will get `http://www.example.com/&ABTEST.`
add your parameter to the end of line above